### ISSUE TITLE
Add support for multiple schemas

### ISSUE THREAD

**Issue by lundberg** [5monkeys/django-bananas.js#79]

**Comment by hbystrom91** [comment]
Right now we get the api instance by referring to BananasAdminContext.api so we would need to either merge the different api(s) into the single api or we would need another way to get access api instances. I guess merging two different apis would come with the risk of having two different endpoints with the same name. I think that would be a minor risk though since the names are prefixed. If I recall correctly the patter usually looks like: `{prefix/namespace}_product_read`.

**Comment by lundberg** [comment]
Agree, probably named apis is the way to go, could help the other parts as well, e.g. named prefix for routes and the top level navigation if thats the way to go.

**Comment by lundberg** [comment]
An alternative is to completely instantiate the admin per schema and make the user login at each api, if not shared auth.

This approach would leave the router and client instance intact since it is per admin app instance.
